css/style.css

```diff
@@ -0,0 +1,130 @@
+body {
+  background-color: #eee;
+}
+
+header {
+  min-height: 75px;
+  position: fixed;
+  display: flex;
+  align-items: center;
+  top: 0;
+  justify-content: space-between;
+  width: 100%;
+  padding: 0px 20px;
+  background-color: #eee;
+}
+
+.logo > img {
+  width: 100%;
+  height: 100%;
+  max-width: 300px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  text-align: center;
+  margin-left: 20px;
+}
```

Comment:
please .. make space between every CSS rule to be used to best CSS practices

Reply:
- [ ] I think you should sound more suggestive. so instead of `please make space` you can say `Kindly consider making space`.

Reply:
- [ ] I think you should sound more suggestive. so instead of please make space you can say Kindly consider making space.

Reply:
- [ ] I think you should sound more suggestive. so instead of please make space you can say Kindly consider making space.